src/main/scala/cromwell/binding/package.scala

```diff
@@ -25,7 +25,7 @@ package object binding {
   type FullyQualifiedName = String
   type LocallyQualifiedName = String
   type CallInputs = Map[String, WdlValue]
-  case class CallOutput(wdlValue: WdlValue, hash: SymbolHash)
+  case class CallOutput(wdlValue: WdlValue, hash: Option[SymbolHash])
```

Comment:
I'll make my usual comment which will be disregarded - that if you have an Option as part of your type, what you really have is a sum type